docker/php-build-stepup/Dockerfile

```diff
@@ -20,19 +17,22 @@ ENV NVM_DIR /usr/local/nvm
 ENV NODE_VERSION 10
 
 # Install nvm with node and npm
-RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh | bash
-    && source $NVM_DIR/nvm.sh \
+RUN mkdir $NVM_DIR \
+    && curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.39.0/install.sh | bash \
+    && . $NVM_DIR/nvm.sh \
+    && ln -s /usr/local/nvm /usr/local/bin/nvm \
     && nvm install $NODE_VERSION \
     && nvm alias default $NODE_VERSION \
-    && nvm use default
+    && nvm use default \
+    # Yarn
+    && npm install --global yarn \
+    # Link Node and Yarn to be available using Docker Run
+    && ln -s /usr/local/nvm/versions/node/v10.24.1/bin/node /usr/local/bin/node \
```

Comment:
Is there no more generic way than a hardcoded version link? This will get outdated quickly

Reply:
Yes, I ended up with exporting the node version to an env-var. And using that instead. That should work out